support/install.sh

```diff
@@ -130,28 +136,26 @@ authenticate() {
 	local PASSWORD=""
 	printf 'Enter username: ' 1>&2
 	read USERNAME
-	while true; do
-		printf 'Enter password: ' 1>&2
-		stty -echo
-		read PASSWORD
-		stty echo
-
-		code=$(curl -w '%{http_code}' \
-			-f -s \
-			-u "${USERNAME}:${PASSWORD}" \
-			-X POST \
-			-o "${INSTALL_DIR}/authz.jwt" \
-			"${SERVER_URL}/api/management/v1/useradm/auth/login")
-		if test $code -eq 200; then
-			break;
-		elif test $code -eq 401; then
-				printf "Authentication failed\n"
-				continue;
-		else
-				printf "Unknown error attempting to login" 2>&1
-				exit 1;
-		fi
-	done
+	printf 'Enter password: ' 1>&2
+	stty -echo
+	read PASSWORD
+	stty echo
+
+	code=$(curl -w '%{http_code}' \
+		-s \
+		-u "${USERNAME}:${PASSWORD}" \
+		-X POST \
+		-o "${INSTALL_DIR}/authz.jwt" \
+		"${SERVER_URL}/api/management/v1/useradm/auth/login")
+	if test $code -eq 200; then
+		break;
```

Comment:
@alfrunes I guess this one doesn't make sense any more without the loop?

Reply:
Ooops, yeah. I removed the loop as it makes more sense to terminate the script if it's the wrong credentials since it might as well be a typo in the email.